### ISSUE TITLE
Connect a VPN profile keeps reconnect when initiating connection

### ISSUE THREAD

**Issue by tommyknchan** [Tunnelblick/Tunnelblick#806]
Hello. Used Tunnelblick for years (Now my macbook OS version is macOS Sonoma), but recently when I use it to connect some Japan OVPN servers, Tunnelbick always shows the connection is keeping repeatedly reconnect when initiating, and never succeed to a proper connection. Tried many OVPN profiles, and even tried on my iPhone, strangely connection is succeed on my phone but never my computer. Please help me to analysis the log attached and how to solve it. Thank you.

[tunnelblick log of ovpn profile.txt](https://github.com/user-attachments/files/15551507/tunnelblick.log.of.ovpn.profile.txt)

**Comment by jkbullard** [comment]
Try different versions of OpenVPN, per [Tunnelblick 4](https://tunnelblick.net/cTunnelblick4.html).

If that doesn't work, contact your VPN service provider for an updated OpenVPN configuration file.

**Comment by tommyknchan** [comment]
Tried the older OVPN version in config, and it works! Connected successfully! Thanks a lot!

**Comment by GitTheHellOutaHere** [comment]
Thanks - worked for me too. 😎

I've been using Viscosity for a long time but decided to give TunnelBlick another try. Immediately it wanted to update to v4.0.1 so I let it. But afterward I was unable to connect to any addresses that Viscosity could connect to.

After more than 30 minutes of diagnosis and searching I stumbled across this topic. When I switched to OpenVPN version 2.5.9 it finally connected.

FWIW I get all my connections [here.](https://www.vpngate.net/en/) For some reason I've never been able to connect to US servers so I exclusively use foreign ones.
